src/cpu/aarch64/acl_batch_normalization.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
-* Copyright 2022 Arm Ltd. and affiliates
+* Copyright 2022, 2024 Arm Ltd. and affiliates
```

Comment:
(nit) usually a `-` character is used to express covering the period.

Reply:
Hi Mourad, thanks for the review. In my understanding the Arm copyright lines follow the convention that `2022-2024` means "The file was modified every year between 2022, and 2024 (e.g. 2022, 2023, **and** 2024)", while the `2022, 2024` above means "The file was only modified in the years 2022, and 2024.". Hope this is clear. This convention is followed in all the copyright headers to the best of my knowledge.